executor.go

```diff
@@ -343,7 +343,16 @@ func (e *Executor) launchTask(taskInfo mesos.TaskInfo) (Command, error) {
 		cmdOption = ForwardCmdOutput()
 	}
 
-	cmd, err := NewCommand(commandInfo, env, cmdOption)
+	beforeStartEvent := hook.Event{
+		Type:     hook.BeforeTaskStartEvent,
+		TaskInfo: mesosutils.TaskInfo{TaskInfo: taskInfo},
+	}
+	additionalEnv, err := e.hookManager.HandleEvent(beforeStartEvent, false)
```

Comment:
You have to merge `env` and `additionalEnv` slices or services will not have access to certificate or `MESOS_` variables.

Reply:
Also add test case for this - the current ones did not catch this problem